### PR TITLE
Adds word vector enrichment

### PR DESCRIPTION
**Pull request by anneschuth**

### CODE REVIEW

File: textpipe/operation.py [hunk 1]
```diff
@@ -70,6 +70,40 @@ def __call__(self, doc):
         return doc.nwords
 
 
+class Words(Operation):
```
Comment (anneschuth) [inline textpipe/operation.py]: I'm trying to sneak this in. Noticed it was missing.

**Comment by anneschuth** [comment]
fair enough, maybe just call it `word_counts`?

File: textpipe/operation.py (hunk 1 above)
Comment (msappelli) [reply textpipe/operation.py]: 👍

File: textpipe/operation.py [hunk 2]
```diff
@@ -89,6 +123,23 @@ def __call__(self, doc):
         return doc.complexity
 
 
+class Sentences(Operation):
```
Comment (anneschuth) [inline textpipe/operation.py]: sneaking this in too, if you allow me

**Comment by JeroenJansze2018** [comment]
Maybe update readme as well?

File: textpipe/doc.py [hunk 3]
```diff
@@ -249,6 +250,18 @@ def words(self):
...
+    def word_counts(self):
+        """
+        Extract words with their counts
+
+        >>> doc = Doc('Test sentence for testing vectorisation of a sentence.')
+        >>> doc.word_counts
+        {'Test': 1, 'sentence': 2, 'for': 1, 'testing': 1, 'vectorisation': 1, 'of': 1, 'a': 1, '.': 1}
+        """
+
+        return dict(Counter(word for word, _ in self.words))
```
Comment (graus) [inline textpipe/doc.py]: This:
```
def s4(mylist):
    mydict={}.fromkeys(mylist,0)
    for k in mydict:
        mydict[k]=mylist.count(k)    
    return mydict
```

or this:
```
def s6(mylist):
    mydict=defaultdict(int)
    for i in mylist:
        mydict[i] += 1
    return mydict     
```
Is supposed to be substantially faster. 

Worth considering, maybe?

Source: https://stackoverflow.com/questions/18343472/efficient-way-to-count-the-element-in-a-dictionary-in-python-using-a-loop

Comment (anneschuth) [reply textpipe/doc.py]: it's not orders of magnitude faster: I'm in favor of clean code.

Comment (graus) [reply textpipe/doc.py]: Fair point, OK

**Pull request merged**